src/playermat/Playmat.ttslua

```diff
@@ -721,7 +663,7 @@ function clickableClues(showCounter)
     local pos = self.positionToWorld({x = -1.12, y = 0.05, z = 0.7})
     for i = 1, clueCount do
       pos.y = pos.y + 0.045 * i
-      spawnToken(pos, "clue")
+      TokenManager.spawnToken(pos, "clue", self.getRotation())
```

Comment:
Playermat rotation is already stored in a variable

Reply:
Changed to be consistent(ish, the file isn't consistent) but I'm not sure this is a good approach.  If the mat were to rotate, this would have an incorrect value.  Pretty unlikely with the playmats, but as a general rule I think anything about object state should be retrieved directly when needed at runtime rather than cached like this